build-image-src/Dockerfile-ruby27

```diff
@@ -50,4 +50,6 @@ RUN pip3 install wheel
 
 ENV LANG=en_US.UTF-8
 
+RUN /sbin/useradd sam
```

Comment:
does this user have sudo priviledges?

Reply:
No, that would require passing `-r`. This is checked by the updated test.

Reply:
Switched to using explicit arguments.